packages/phrases/src/locales/en.ts

```diff
@@ -66,6 +66,7 @@ const errors = {
     insufficient_info: 'Insufficent sign-in info.',
     connector_id_mismatch: 'The connectorId is mismatched with session record.',
     connector_session_not_found: 'Connector session not found. Please go back and sign in again.',
+    unauthorized: 'Please sign in first.',
```

Comment:
nit: please sign in before binding?

Reply:
Other actions may require sign in as well.